src/KafkaFlow.Abstractions/ISerializationContext.cs

```diff
@@ -0,0 +1,9 @@
+namespace KafkaFlow
```

Comment:
IMHO, `ISerializerContext` would fit better since this is the context used in the Serializers.

Reply:
Agreed!